### PR TITLE
[ci] Add CI check for migrations

### LINKED ISSUE #12: [ci] Add CI check for migrations
Add CI check for migrations with [this solution](https://github.com/openwisp/openwisp-users/pull/35/files) for ensuring the migrations created have have been renamed to something more descriptive like [this GCI task says](https://codein.withgoogle.com/dashboard/task-instances/5806059705860096/).

### PR DESCRIPTION
**Pull request by marfgold1**
For ensuring the migrations created
have have been renamed to something
more descriptive

Closes #12

### CODE REVIEW

**Comment by marfgold1** [comment]
Anyway thanks for @dwang and @EdgeKing810
Please review my PR, @atb00ker or @nemesisdesign, thank you.

File: runcheckmigration [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+set -e
+checkmigrations --migration-path ./django_loci/migrations/ || exit 1
```
Comment (atb00ker) [inline runcheckmigration]: Add an empty line at the end of a file. (Always add an empty line at the end of the file)

Comment (marfgold1) [reply runcheckmigration]: done 👍

**Comment by marfgold1** [comment]
Whoops, been missed that one, sorry. I thought there is only one, so i'm not using the find file to recheck all folder named `migrations`, my bad. I'll be more careful after this. Thanks anyway @atb00ker ! Sorry for the inconvenience, though 😞

**Pull request merged**